ui/imports/shared/popups/SendModal.qml

```diff
@@ -229,6 +227,9 @@ StatusDialog {
                                 }
                                 return ""
                             }
+                            getNetworkIcon: function(chainId){
```

Comment:
I agree, maybe we need to restructure the assets model to also hold this information. Let me create a separate task for it, cause it will affect a lot of places in wallet. 

https://github.com/status-im/status-desktop/issues/8923